src/Migration/EmailGatewayMigration.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Terminal42\NotificationCenterBundle\Migration;
+
+use Contao\CoreBundle\Migration\AbstractMigration;
+use Contao\CoreBundle\Migration\MigrationResult;
+use Doctrine\DBAL\Connection;
+
+class EmailGatewayMigration extends AbstractMigration
+{
+    public function __construct(private readonly Connection $connection)
+    {
+    }
+
+    public function shouldRun(): bool
+    {
+        return $this->connection->fetchOne("SELECT COUNT(*) FROM tl_nc_gateway WHERE type='email'") > 0;
```

Comment:
Won't this fail when installing NC for the first time?

Reply:
@aschempp that's probably true, will you create another PR?

Reply:
Thanks @fritzmg, fixed in ba534a3a20f11fabb5105d7d7d1c87a68bf54927